harness/determined/common/experimental/trial.py

```diff
@@ -353,6 +372,13 @@ def key(ckpt: checkpoint.Checkpoint) -> Any:
     def __repr__(self) -> str:
         return "Trial(id={})".format(self.id)
 
+    def _stream_metrics(self, group: str) -> Iterable[_TrialMetrics]:
```

Comment:
why is this wrapper method needed? why not just call `_stream_trials_metrics` directly?

Reply:
just following existing patterns to avoid more change than necessary. see stream_training_metrics